extension/extension/background.js

```diff
@@ -9,7 +9,8 @@ var RAW_DEBUG = false;
 /* This variable must be manually updated to specify the server that
  * the data will be sent to.  
 */
-var WEBSOCKET_SERVER_URL = "wss://learning-observer.org/wsapi/in/" 
+var WEBSOCKET_SERVER_URL = "wss://gayaan.csc.ncsu.edu/wsapi/in/"
+// var WEBSOCKET_SERVER_URL = "ws://localhost:8888/wsapi/in/"
```

Comment:
We don't want lines like this in commits. 

I believe you might be able to override the server with a setting in local storage, although somehow the code to use this is commented out, so it might be necessary to fix it. I suspect this might be related to initialization logic (things are async, so they get a bit complex).